test/README.md

```diff
@@ -106,34 +96,55 @@ be documented.
 ### API Tests (`test/bdd/api`)
 
 These tests are meant to test the different API endpoints and their parameters.
-They require a preimported test database, which consists of the import of a
-planet extract. A precompiled PBF with the necessary data can be downloaded from
-https://www.nominatim.org/data/test/nominatim-api-testdata.pbf
+They require a to import several datasets into a test database. You need at
+least 2GB RAM and 10GB discspace.
 
-You need at least 2GB RAM and 10GB discspace.
 
-The polygons defining the extract can be found in the test/testdb
-directory. There is also a reduced set of wikipedia data for this extract,
-which you need to import as well. For Tiger tests the data of South Dakota
-is required. Get the Tiger files `46*`.
+1. Fetch the OSM planet extract (200MB). See end of document how it got created.
 
+    ```
     cd Nominatim/data
+    mkdir testdb
+    cd testdb
+    wget https://www.nominatim.org/data/test/nominatim-api-testdata.pbf
+    ```
+
+2. Fetch `46*` (South Dakota) Tiger data
+
+    ```
+    cd Nominatim/data/testdb
     wget https://nominatim.org/data/tiger2018-nominatim-preprocessed.tar.gz
     tar xvf tiger2018-nominatim-preprocessed.tar.gz --wildcards --no-anchored '46*'
     rm tiger2018-nominatim-preprocessed.tar.gz
+    ```
 
-The official test dataset is derived from the 180924 planet. Newer
-planets are likely to work as well but you may see isolated test
-failures where the data has changed. To recreate the input data
-for the test database run:
+3. Use a reduced set of wikipedia data
 
-    wget https://ftp5.gwdg.de/pub/misc/openstreetmap/planet.openstreetmap.org/pbf/planet-180924.osm.pbf
-    osmconvert planet-180924.osm.pbf -B=test/testdb/testdb.polys -o=testdb.pbf
+    ```
+    cd Nominatim/data/testdb/
+    cp ../../test/testdb/wikimedia-importance.sql.gz .
+	gunzip wikimedia-importance.sql.gz
+    ```
```

Comment:
This step is not necessary. Simply set

    @define('CONST_Wikipedia_Data_Path', CONST_BasePath.'/test/testdb');

below.